rocAL/rocAL/include/api/rocal_api_types.h

```diff
@@ -41,6 +41,8 @@ typedef void * RocalIntParam;
 typedef void * RocalContext;
 typedef void * RocalImage;
 typedef void * RocalMetaData;
```

Comment:
Do we need to keep typedef void * RocalMetaData for now ?

Reply:
Will keep it for now